### ISSUE TITLE
Do we want to display countries that don't have specific goods listed?

### ISSUE THREAD

**Issue by caseymm** [ahlusar1989/ILAB_VIZ_demo#1]
![screen shot 2015-05-31 at 8 51 59 am](https://cloud.githubusercontent.com/assets/3526984/7901991/59b89dea-0772-11e5-9f3a-ba6c2ae97b42.png)

**Comment by tyronegrandison** [comment]
Yes. Some will be grayed out because they have not been assessed. Some will
contain data (I.e. description is not empty) but no goods because either
they have services that contain forced or child labor or no goods have been
officially confirmed.

**Comment by caseymm** [comment]
Ok, that works. Where are the services listed? Didn't see them listed the same away that goods are listed?

Along those lines, should we also be showing services overview on the main landing page (like we are for goods currently)?

**Comment by tyronegrandison** [comment]
The services are listed in Table 2.

There is no standard set of services. :-(
